packages/demo/src/components/examples/BannerExamples.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import {Banner, BannerContainer, BannerMessageStates} from 'react-vapor';

```

Comment:
You can also delete the Banner & BannerMessageStates since they aren't used anywhere in the admin